enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
I would prefer to follow http://semver.org since that is the generally-accepted approach for other edX packages.  We need to at least increase the patch with every change because this is how the package is referenced in edx-platform.